include/effects_internal.h

```diff
@@ -3,6 +3,12 @@
 
 #include "effects.h"
 
+#ifdef PRINT_ON_PANIC
+// TODO: provide shim for func_80025F44 instead
```

Comment:
what's that function do? what does this change have to do with fixing --debug

Reply:
See macros.h